apps/fabric-website/src/root.tsx

```diff
@@ -12,6 +12,8 @@ import { HomePage } from './pages/HomePage/HomePage';
 import WindowWidthUtility from './utilities/WindowWidthUtility';
 import './styles/styles.scss';
 import { initializeIcons } from 'office-ui-fabric-react/lib/Icons';
+import 'whatwg-fetch';
```

Comment:
was this intentional? I don't see the use of it added below.

Reply:
thought I needed this to use the Fetch API but turns out I don't. Removing it.